src/components/DocumentInfo/AddressMultiple/AddressMultiple.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
```

Comment:
I'm not sure about the naming AddressSimple / AddressMultiple

Maybe can refer to the intent of the feature ?

I would refer AddressSimple just as Address, but for AddressMultiple I'm not sure and dont feel I can help :(

Reply:
renamed to TokenAddressesView lol